### PR TITLE
New comp

### PR DESCRIPTION
**Pull request by webbdm**
@sbussard @joelwalden 

Thoughts?

My plan is continuing to tweak the styling & alignments some more before adding interactivity to any of the components & wiring it up to Redux.

### CODE REVIEW

File: src/Camp.css [hunk 1]
```diff
@@ -1,35 +1,171 @@
-.Center {
-  display: block;
-  width: 50%;
-  margin: 70px auto;
+.Wrapper {
```
Comment (sbussard) [inline src/Camp.css:1]: I suggest you use this convention for naming your CSS elements
https://suitcss.github.io

File: src/Camp.css [hunk 2]
```diff
@@ -1,35 +1,171 @@
...
+  flex-direction:row;
+  flex-wrap: wrap;
   text-align: center;
-  background-color: #2C7E9A;
-  max-width: 450px;
-  border: 3px solid gray;
-  border-radius: 12px;
+  align-content: space-between;
+  background-color: #292727;
+  max-width: 1080px;
```
Comment (sbussard) [inline src/Camp.css]: Use a combination of rems and ems, avoid px.
https://css-tricks.com/building-resizeable-components-relative-css-units

File: src/Camp.css [hunk 3]
```diff
@@ -1,35 +1,171 @@
...
+  display: flex;
+  flex-direction:row;
+  flex-wrap: wrap;
   text-align: center;
-  background-color: #2C7E9A;
-  max-width: 450px;
-  border: 3px solid gray;
-  border-radius: 12px;
+  align-content: space-between;
+  background-color: #292727;
```
Comment (sbussard) [inline src/Camp.css]: If you use SCSS you can specify your color palette in a series of variables. 
That way you ensure that your colors are consistent over time.

File: src/Profile.js [hunk 4]
```diff
@@ -0,0 +1,93 @@
+import React, { Component } from 'react';
+//import logo from './logo.svg';
```
Comment (sbussard) [inline src/Profile.js]: Remove comments

File: src/Profile.js [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+const Resume = (props) => 
+      <div className="resume">
+         <div className="nav">{/*GEOFF <span>WEB</span>B*/}</div>
+         <div className="my-info">
+            <MyInfo/>
+         </div>
+         <div className="experience">
+            <MyXP/>
+         </div>
+            <Camp jobs={props.jobs}/>
```
Comment (sbussard) [inline src/Profile.js:21]: What is the semantic meaning of `Camp`?

File: src/Profile.js [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+        <img src="https://media.licdn.com/media/p/2/005/02a/1c1/0e167c8.jpg" alt="Geoff"/>
+        <h3></h3>
+        <div className="logoDiv">
+          <img src="https://angular.io/resources/images/logos/angular2/angular.svg" alt="Angular"/>
+          <img src="http://tomwayson.github.io/mwd-jsapi/img/react-logo.png" alt="React"/>
+          <img src="https://dt-cdn.net/assets/images/gfx/icons/tech/dotnetmicrosoft-536938fee2.svg" alt=".NET"/>
+        </div>
+
+        <div className="infoDiv">
+          <h4>Education</h4>
```
Comment (sbussard) [inline src/Profile.js:52]: Avoid hard-coding your copy. 
Probably pull it in from a json or yml file

**Review by sbussard — COMMENTED** [review]
I left a few suggestions. 

It'd be good to get into the habit of making smaller / more-focused pull requests that focus on one thing at a time. 

I recommend you spend time considering how you want data to get into components. It's ok to pull information in from a json file rather than have it passed in via props. The main reason you'd want to pass information in via props is if you want that particular component to be highly reusable. You probably won't have multiple resumes, but you'll probably want to have multiple rows on your resume. It would make sense for your top level "Resume" component to pull in the information from a separate file and pass parts of that info into smaller components for your education items, experience items, contact information, etc.. 

I hope that makes sense.

**Comment by sbussard** [comment]
Also, don't use redux for this. It's way too complicated to get bogged down with it at this time

File: src/Profile.js (hunk 5 above)
Comment (webbdm) [reply src/Profile.js:21]: Yeah I need to change that ASAP. The project started under a different name ¯\\\_(ツ)_/¯

File: src/Profile.js (hunk 6 above)
Comment (webbdm) [reply src/Profile.js:52]: :+1:

**Comment by webbdm** [comment]
Yeah it's overkill on a project this small, was just gonna implement it at a later time for the sake of getting the hang of it.

File: src/Profile.js (hunk 4 above)
Comment (webbdm) [reply src/Profile.js]: Oops remnants from npm react-create-app

**Pull request merged**